slackclient/_server.py

```diff
@@ -138,7 +140,7 @@ def send_to_websocket(self, data):
         try:
             data = json.dumps(data)
             self.websocket.send(data)
-        except:
+        except websocket.WebSocketException:
```

Comment:
I'd almost rather not have this library doing reconnects itself, but c'est la vie